bigframes/ml/imported.py

```diff
@@ -134,7 +138,7 @@ def _create_bqml_model(self):
     def _from_bq(cls, session: bigframes.Session, model: bigquery.Model) -> ONNXModel:
         assert model.model_type == "ONNX"
 
-        onnx_model = cls(session=session, model_path=None)
+        onnx_model = cls(session=session, model_path="")
```

Comment:
Curious what difference this makes ("" over None)

Reply:
Nothing, if loaded from an existing model, we don't have a way to retrieve the model file path, as the model has already been loaded. Both None and "" are placeholders, change the value only to satisfy mypy, that we changed the parameter to be non-optional.